applications/LS-Dyna-stampede3/app.json

```diff
@@ -0,0 +1,124 @@
+{
+    "id": "LS-Dyna-stampede3",
+    "version": "2024R1",
+    "description": "LS-DYNA is an advanced general-purpose multiphysics simulation software package.",
+    "owner": "${apiUserId}",
+    "enabled": true,
+    "runtime": "ZIP",
+    "runtimeVersion": null,
+    "runtimeOptions": null,
+    "containerImage": "tapis://cloud.data//corral/tacc/aci/CEP/applications/v3/ls-dyna-stampede3/ls-dyna-stampede3.zip",
+    "jobType": "BATCH",
+    "maxJobs": -1,
+    "maxJobsPerUser": -1,
+    "strictFileInputs": true,
+    "jobAttributes": {
+        "description": "",
+        "dynamicExecSystem": false,
+        "execSystemConstraints": null,
+        "execSystemId": "stampede3",
+        "execSystemExecDir": "${JobWorkingDir}",
+        "execSystemInputDir": "${JobWorkingDir}",
+        "execSystemOutputDir": "${JobWorkingDir}",
+        "execSystemLogicalQueue": "skx-dev",
+        "archiveSystemId": "cloud.data",
+        "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+        "archiveOnAppError": true,
+        "isMpi": false,
+        "mpiCmd": null,
+        "cmdPrefix": null,
+        "parameterSet": {
+            "appArgs": [
+                {
+                    "name": "Input file",
+                    "description": "Please enter the input file in .txt format. Add one .k case file per line if there are multiple case files.",
+                    "arg": "myfilename.txt",
```

Comment:
Is this a valid default arg? If not, you can leave blank.

Reply:
This is valid. The case files name are included in the .txt file.